### PR TITLE
Allow additional params for list_transactions

### PR DESCRIPTION
**Pull request by patrickarnett**
Allow use of `count`, `order`, and `state` params.

### CODE REVIEW

**Comment by jeremywrowe** [comment]
@patrickarnett - We just added a changelog, would you mind adding your contributions to it in this pull request?

**Comment by patrickarnett** [comment]
I've updated the PR.

**Comment by patrickarnett** [comment]
@jeremywrowe I've updated the PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,3 +16,4 @@ All notable changes to this project will be documented in this file.
 - @adamcohen - Expose `transaction_type` on transaction
 - @adamcohen - Expose `gateway_specific_response_fields` on transaction
 - @adamcohen - Expose `payment_method_type` on transaction
+- @patrickarnett - Modify `list_transactions` and `list_transactions_url` to take `count`, `order`, and `state` arguments in an optional hash.
```
Comment (jeremywrowe) [inline CHANGELOG.md]: This should go in the `unreleased` section (right below line 5) and we'll pull it down into a release version when we cut a release.

Comment (patrickarnett) [reply CHANGELOG.md]: Oops!

**Comment by jeremywrowe** [comment]
Thanks @patrickarnett - I've merged the pull request. I'll be cutting a release in the next week or so.

**Pull request merged**